shared/js/project.js

```diff
@@ -269,6 +269,9 @@ module.exports = {
       executeCallback(callback);
     });
   },
+  serverSideRemix: function() {
+    location.href = module.exports.getPathName('remix');
```

Comment:
Do you think it would be better if line 48 was var projects = module.exports = { ... }? Then here we can do projects.getPathName instead of module.exports.getPathName. Not sure whether I feel strongly about this.